config/gardener-certmanager/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+resources:
+  - crd/dns.gardener.cloud_dnsentries.yaml # needed for GCM to work since this is not included in the helm chart for GCM
```

Comment:
I didn't get it, why we need to this 3rd party CRD, shouldn't GCM provide this by itself if it's installed in the KCP in the future?

it not included in the helm chart on purpose or it's their design oversight? I mean it's anyway not a good idea provide by the consumer, if we go down that path, how can we ensure compatibility between the CRD and GCM, What happens if a newer GCM version no longer supports the current CRD version, How would we handle upgrades safely, those question should be clarified first.

Even if we have to do it, providing the CRD as a static local file is not an acceptable solution, we lose visibility and traceability of the source.